gammapy/estimators/points/core.py

```diff
@@ -336,8 +341,15 @@ def from_table(
         """
         table = table_standardise_units_copy(table)
 
-        if format is None:
-            format = cls._table_guess_format(table)
+        guessed_format = cls._table_guess_format(table)
+
+        if format is not None and format != guessed_format:
```

Comment:
Is this necessary? The user can have the liberty to override the guessed format.

Reply:
Yes perhaps this check is too harsh